core/opl/shovel.py

```diff
@@ -119,15 +316,93 @@ def args(parser, group_actions):
             title="horreum",
             description="Options needed to work with Horreum",
         )
+        group.add_argument("--horreum-data-file", help="Data file to upload to Horreum")
+        group.add_argument("--horreum-host", help="Horreum host url")
+        group.add_argument("--token", help="Authorisation token")
+        group.add_argument("--job-name", help="Job name")
+        group.add_argument(
+            "--test-name-horreum", default="load-tests-result", help="Test Name"
+        )
+        group.add_argument("--test-job-matcher", default="jobName")
+        group.add_argument("--test-owner", default="rhtap-perf-test-team")
+        group.add_argument("--test-access", default="PUBLIC")
+        group.add_argument(
+            "--id-array", help="Variable id array we wish to get timeseries data for"
```

Comment:
What it is actually for? Could you point me for some docs somewhere? Maybe add more useful help text, with this I feel bit lost :) Can the user maybe just provide coma separated list of IDs instead of JSON?